src/tests/baseservices/callconvs/CallFunctionPointers.il

```diff
@@ -48,6 +48,33 @@
     ret
   }
 
+  .method public hidebysig static int32 CallUnmanagedIntInt_ModOptCdecl(void* p, int32 b) cil managed
```

Comment:
I would also like to test the behavior if we have multiple `modopt` values as well as ensure we handle the case when we have a a clearly defined `unmanaged *` with a different `modopt` (e.g. `calli unmanaged stdcall int32 modopt([System.Runtime]System.Runtime.CompilerServices.CallConvCdecl) (int32)`)